.github/workflows/pull_request.yml

```diff
@@ -20,6 +20,8 @@ jobs:
           wget -P $M2_HOME/lib/ext https://repo1.maven.org/maven2/io/takari/aether/takari-local-repository/0.11.3/takari-local-repository-0.11.3.jar
           wget -P $M2_HOME/lib/ext https://repo1.maven.org/maven2/io/takari/takari-filemanager/0.8.3/takari-filemanager-0.8.3.jar
           wget -P $M2_HOME/lib/ext https://repo1.maven.org/maven2/io/takari/maven/takari-smart-builder/0.6.1/takari-smart-builder-0.6.1.jar
+        env:
+          M2_HOME: /usr/share/apache-maven-3.6.3
```

Comment:
Hard-coded Maven version. This will break once Maven is upgraded in the image used to run the workflow. See https://issues.redhat.com/browse/BXMSPROD-1214?focusedCommentId=15837371&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-15837371 for more details and an alternative solution.

Reply:
Many thanks @yurloc 
Check my comment here
I propose to improve it on https://issues.redhat.com/browse/BXMSPROD-1216